src/lib/radio/radio.ts

```diff
@@ -125,7 +125,16 @@ export class MatRadioGroup extends _MatRadioGroupMixinBase
    * Change events are only emitted when the value changes due to user interaction with
    * a radio button (the same behavior as `<input type-"radio">`).
    */
-  @Output() change: EventEmitter<MatRadioChange> = new EventEmitter<MatRadioChange>();
+  @Output() selectionChange: EventEmitter<MatRadioChange> = new EventEmitter<MatRadioChange>();
+
+  /**
+   * Event emitted when the group value changes.
+   * Change events are only emitted when the value changes due to user interaction with
+   * a radio button (the same behavior as `<input type-"radio">`).
+   * @deprecated use `selectionChange` instead.
+   */
+  @Output() change: EventEmitter<MatRadioChange> = this.selectionChange;
```

Comment:
To aide in eventually removing `change` `()Output` in the future, can you please update the references within the `MatRadioGroup` to call for `selectionChange` instead of the `change`?

Reply:
Updated